src/Gui/Windows/CombinedCodeViewInteractor.cs

```diff
@@ -299,12 +304,39 @@ public bool ChooseTextEncoding()
             return true;
         }
 
-        private void MixedCodeDataView_MouseDown(object sender, EventArgs e)
+        private void MixedCodeDataView_MouseUp(object sender, MouseEventArgs e)
+        {
```

Comment:
I don't want to lose the ability to use hungarian to quickly denote the type of a range of memory. It's faster for me to  select a memory range and type `apsz` (array of pointers to zero-terminated strings) than `char * foo [124]`.

I suggest you continue working on the CombinedCodeView until you reach a stage where it is reasonably complete, and in the meantime I will give some thought to how these could be combined.